packages/drawer/src/LionDrawer.js

```diff
@@ -0,0 +1,245 @@
+   * @override
+   */
+  async _hideAnimation({ contentNode }) {
+    if (
+      ((this.position === 'left' || this.position === 'right') &&
+        this._contentWidth === this.minWidth) ||
+      (this.position === 'top' && this._contentHeight === this.minHeight)
+    ) {
+      return;
+    }
+
+    const min = this.position === 'top' ? this.minHeight : this.minWidth;
+    const prop = this.position === 'top' ? 'height' : 'width';
+
+    contentNode.style.setProperty(prop, /** @type {string} */ (min));
+    await this._waitForTransition({ contentNode });
+  }
+
+  /**
+   *  Wait until the transition event is finished.
+   * @param {Object} options - element node and its options
+   * @param {HTMLElement} options.contentNode
+   * @returns {Promise<void>} transition event
+   */
+  _waitForTransition({ contentNode }) {
+    return new Promise(resolve => {
+      const transitionStarted = () => {
+        contentNode.removeEventListener(EVENT.TRANSITION_START, transitionStarted);
+        this.transitioning = true;
+      };
+      contentNode.addEventListener(EVENT.TRANSITION_START, transitionStarted);
+
+      const transitionEnded = () => {
+        contentNode.removeEventListener(EVENT.TRANSITION_END, transitionEnded);
+        this.transitioning = false;
+        resolve();
+      };
+      contentNode.addEventListener(EVENT.TRANSITION_END, transitionEnded);
+    });
+  }
+
+  /**
+   * @protected
+   */
+  get _bottomInvokerNode() {
+    return /** @type {HTMLElement[]} */ (Array.from(this.children)).find(
+      child => child.slot === 'bottom-invoker',
+    );
+  }
+
+  /**
+   * @protected
+   */
+  get __contentNode() {
+    return /** @type {HTMLElement} */ (this.shadowRoot?.querySelector('.container'));
+  }
+
+  get _contentWidth() {
+    const size = this.__contentNode?.getBoundingClientRect().width || 0;
+    return `${size}px`;
+  }
+
+  get _contentHeight() {
+    const size = this.__contentNode?.getBoundingClientRect().height || 0;
+    return `${size}px`;
+  }
+
+  _openedChanged() {
+    this._updateContentSize();
+    if (this._invokerNode) {
+      this._invokerNode.setAttribute('aria-expanded', `${this.opened}`);
+    }
+
+    if (this._bottomInvokerNode) {
+      this._bottomInvokerNode.setAttribute('aria-expanded', `${this.opened}`);
+    }
+
+    this.dispatchEvent(new CustomEvent('opened-changed'));
+  }
+
+  async _updateContentSize() {
+    if (this.__contentNode) {
+      if (this.opened) {
+        await this._showAnimation({ contentNode: this.__contentNode });
+      } else {
+        await this._hideAnimation({ contentNode: this.__contentNode });
+      }
+    }
+  }
+
+  render() {
+    return html`
+      <div class="container">
+        <div class="headline-container">
+          <slot name="invoker"></slot>
+          <slot name="headline"></slot>
+        </div>
+        <div class="content-container">
+          <slot name="content"></slot>
+        </div>
+        <slot name="bottom-invoker"></slot>
```

Comment:
Not sure about the need of 2 invoker slots in lion. It is intended to be extend the component, and normally speaking only 1 invoker is needed. They can override the template to change the order (content first and invoker later).
Makes the code much simpler too.

Reply:
I checked again with designers and we don't need them at the top and bottom simultaneously so I will remove this and to the implementation for it in ing-web.

Reply:
fixed